lib/Selenium/Client.pm

```diff
@@ -136,8 +137,11 @@ sub new($class,%options) {
     $options{auto_close} //= 1;
     $options{browser}    //= '';
 
+    #create client_dir and log-dir
+    my $dir = File::Spec->catdir( $options{client_dir},"perl-client" );
+    make_path($dir);
```

Comment:
Thanks for this -- I was relying on the make_path in Selenium::Specification which is perhaps too tight of a coupling.